src/platform_impl/windows/util.rs

```diff
@@ -72,80 +61,80 @@ pub fn get_instance_handle() -> windows_sys::Win32::Foundation::HMODULE {
-            // this DPI value.
-            GetDeviceCaps(hdc, LOGPIXELSX as _) as u32
-        } else {
-            // If the process is DPI unaware, then scaling is performed by the OS; we thus return
-            // 96 (scale factor 1.0) to prevent the window from being re-scaled by both the
-            // application and the WM.
-            BASE_DPI
-        }
-    }
-}
+// fn get_function_impl(library: &str, function: &str) -> FARPROC {
+//     let library = encode_wide(library);
+//     assert_eq!(function.chars().last(), Some('\0'));
+
+//     // Library names we will use are ASCII so we can use the A version to avoid string conversion.
+//     let module = unsafe { LoadLibraryW(library.as_ptr()) };
+//     if module == 0 {
+//         return None;
+//     }
+
+//     unsafe { GetProcAddress(module, function.as_ptr()) }
+// }
+
+// macro_rules! get_function {
+//     ($lib:expr, $func:ident) => {
+//         crate::platform_impl::platform::util::get_function_impl(
+//             $lib,
+//             concat!(stringify!($func), '\0'),
+//         )
+//         .map(|f| unsafe { std::mem::transmute::<_, $func>(f) })
+//     };
+// }
+
+// pub type GetDpiForMonitor = unsafe extern "system" fn(
+//     hmonitor: HMONITOR,
+//     dpi_type: MONITOR_DPI_TYPE,
+//     dpi_x: *mut u32,
+//     dpi_y: *mut u32,
+// ) -> HRESULT;
+
+// static GET_DPI_FOR_MONITOR: Lazy<Option<GetDpiForMonitor>> =
+//     Lazy::new(|| get_function!("shcore.dll", GetDpiForMonitor));
+
+// pub const BASE_DPI: u32 = 96;
+// pub fn dpi_to_scale_factor(dpi: u32) -> f64 {
+//     dpi as f64 / BASE_DPI as f64
+// }
+
+// #[allow(non_snake_case)]
+// pub unsafe fn hwnd_dpi(hwnd: HWND) -> u32 {
+//     if let Some(GetDpiForMonitor) = *GET_DPI_FOR_MONITOR {
+//         // We are on Windows 8.1 or later.
+//         let monitor = MonitorFromWindow(hwnd, MONITOR_DEFAULTTONEAREST);
+//         if monitor == 0 {
+//             return BASE_DPI;
+//         }
+
+//         let mut dpi_x = 0;
+//         let mut dpi_y = 0;
+//         #[allow(clippy::unnecessary_cast)]
+//         if GetDpiForMonitor(monitor, MDT_EFFECTIVE_DPI, &mut dpi_x, &mut dpi_y) == S_OK {
+//             dpi_x as u32
+//         } else {
+//             BASE_DPI
+//         }
+//     } else {
+//         let hdc = GetDC(hwnd);
+//         if hdc == 0 {
+//             return BASE_DPI;
+//         }
+
+//         // We are on Vista or later.
+//         if IsProcessDPIAware() == 1 {
+//             // If the process is DPI aware, then scaling must be handled by the application using
+//             // this DPI value.
+//             GetDeviceCaps(hdc, LOGPIXELSX as _) as u32
+//         } else {
+//             // If the process is DPI unaware, then scaling is performed by the OS; we thus return
+//             // 96 (scale factor 1.0) to prevent the window from being re-scaled by both the
+//             // application and the WM.
+//             BASE_DPI
+//         }
+//     }
+// }
```

Comment:
Not sure if we need to keep these

Reply:
Just remove, we can add back from commit history if needed